src/utils/api.js

```diff
@@ -10,13 +10,16 @@ const {
   AFTERNOON,
   EVENING,
   NIGHT,
+  LIVE_VIEW_STRING,
+  COMPOSE_VIEW_STRING,
 } = require('../config/constants');
 
 // Takes array of action objects and returns an object with {key: value} pairs of {date: #-of-actions}
-export const getActionsByDay = (actions) => {
+export const getActionsByDay = (actions, view) => {
+  const dateKey = view === COMPOSE_VIEW_STRING ? 'published' : 'createdAt';
   const actionsByDay = {};
   actions.forEach((action) => {
-    const actionDate = new Date(action.createdAt.slice(0, 10));
+    const actionDate = new Date(action[dateKey].slice(0, 10));
```

Comment:
Didn't you write a function to that wrapped this `slice` operation with a name like `extractDateFromTimestamp` or something like that?

Reply:
so... I believe we talked about that for `getActionHourOfDay` (which we used in a more recent update), where I am also slicing to get the hour of day. let me take care of these in a separate issue/PR (maybe using moment.js)